addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml

```diff
@@ -176,4 +228,77 @@
 		<state readOnly="false"/>
     </channel-type>
 
+    <channel-type id="actual_temp" advanced="true">
+        <item-type>Number</item-type>
+        <label>Actual Temperature of heating thermostat</label>
```

Comment:
Labels should be as short as possible, usually 1-2 words only (need to fit on smartphone screens).
So better reduce it to "Actual Temperature"